src/renderer/components/+landing-page/landing-page.tsx

```diff
@@ -11,28 +11,28 @@ import { Icon } from "../icon";
 
 @observer
 export class LandingPage extends React.Component {
-  @observable showHint = true;
+  @computed get workspace() {
+    return workspaceStore.currentWorkspace;
+  }
 
-  @computed
-  get clusters() {
-    return clusterStore.getByWorkspaceId(workspaceStore.currentWorkspaceId);
+  @computed get workspaceClusters() {
+    return clusterStore.getByWorkspaceId(this.workspace.id);
   }
 
   componentDidMount() {
-    const noClustersInScope = !this.clusters.length;
-    const showStartupHint = this.showHint;
-
-    if (showStartupHint && noClustersInScope) {
+    if (!workspaceStore.hasBeenSeen(this.workspace) && this.workspaceClusters.length === 0) {
       Notifications.info(<><b>Welcome!</b><p>Get started by associating one or more clusters to Lens</p></>, {
         timeout: 30_000,
         id: "landing-welcome"
       });
     }
+
+    workspaceStore.markSeen(this.workspace);
   }
 
   render() {
-    const showBackButton = this.clusters.length > 0;
-    const header = <><Icon svg="logo-lens" big /> <h2>{workspaceStore.currentWorkspace.name}</h2></>;
+    const showBackButton = this.workspaceClusters.length > 0;
```

Comment:
I think the back button should only be shown if there is an active cluster in this workspace because this is the landing page, there is no place to go back to from the landing page if there is no active cluster. (unless we go to the previous workspace, if there is one. But I think that would be a weird UX)